static/app/views/issueDetails/groupOpenPeriods.tsx

```diff
@@ -0,0 +1,104 @@
+import {useState} from 'react';
+import styled from '@emotion/styled';
+
+import {DateTime} from 'sentry/components/dateTime';
+import Duration from 'sentry/components/duration';
+import GridEditable, {
+  COL_WIDTH_UNDEFINED,
+  type GridColumnOrder,
+} from 'sentry/components/gridEditable';
+import LoadingError from 'sentry/components/loadingError';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {t} from 'sentry/locale';
+import {useParams} from 'sentry/utils/useParams';
+import {useGroup} from 'sentry/views/issueDetails/useGroup';
+
+type OpenPeriodDisplayData = {
+  duration: React.ReactNode;
+  end: React.ReactNode;
+  start: React.ReactNode;
+  title: React.ReactNode;
+};
+
+// TODO(snigdha): make this work for the old UI
+// TODO(snigdha): support pagination
+function IssueOpenPeriodsList() {
+  const [now] = useState(() => new Date());
+  const params = useParams<{groupId: string}>();
+  const {
+    data: group,
+    isPending: isGroupPending,
+    isError: isGroupError,
+    refetch: refetchGroup,
+  } = useGroup({groupId: params.groupId});
+
+  if (isGroupError) {
+    return <LoadingError onRetry={refetchGroup} />;
+  }
+
+  if (isGroupPending) {
+    return <LoadingIndicator />;
+  }
+
+  // update the open periods to have date objects
+  const openPeriods = group?.openPeriods?.map(period => ({
+    ...period,
+    start: new Date(period.start),
+    end: period.end ? new Date(period.end) : null,
+  }));
```

Comment:
group should be defined now that you've moved  the loading states above
```suggestion
  const openPeriods = group.openPeriods?.map(period => ({
    ...period,
    start: new Date(period.start),
    end: period.end ? new Date(period.end) : null,
  }));
```